proposals/Exceptions.md

```diff
@@ -306,14 +306,31 @@ or defined:
 
 ### Exception section
 
-The `exception` section is the named section 'exception'. The exception section
-declares exception types using exception type signatures.
+The `exception` section is the named section 'exception' with id 13. The
+exception section may occur anywhere after the import section and before the
```

Comment:
The order of all known sections is fixed. We shouldn't lose that property. For maximum future-proofness I suggest inserting it between import section (2) and function section (3).